CONTRIBUTING.md

```diff
@@ -17,3 +21,48 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 All contributions to this project will be released under the CC0
 dedication. By submitting a pull request, you are agreeing to comply
 with this waiver of copyright interest.
+
+## Bugs
+
+### Known Issues
+
+We track issues on the GitHub [Issues page](https://github.com/ustaxcourt/ef-cms/issues). Before filing any new issues, please check to see if we've addressed the problem before or already have an existing issue related to it.
+
+### Reporting Issues
+
+From the [Issues page](https://github.com/ustaxcourt/ef-cms/issues) you can click the `New issue` button or go straight to the [list of templates](https://github.com/ustaxcourt/ef-cms/issues/new/choose).
+
+### Security-related Issues
+
+For security-sensitive issues, please do not file a public issue. Rather, please email [Dawson support](dawson.support@ustaxcourt.gov).
+
+## Proposing Changes
+
+We welcome pull requests, but we do still want those changes to be related to an [Issue](#reporting-issues). Please file one prior to submitting a pull request.
+
+## Pull Request Procedures
+
+Please follow these steps before submitting a pull request:
+
+1. Fork the repository and create a branch from `staging`.
+1. Run `npm install` in the repository root.
+   1. Do not delete `package-lock.json`.
+1. If reasonable for the changes being made, write tests.
+1. Run `npm run lint:fix` in the repository root.
+1. Ensure that all tests run with the following scripts pass:
```

Comment:
We might want to restate the following line:  "Ensure that all tests run with the following scripts pass"
Might be better to say:  "Ensure that all tests run and that the following scripts pass"

Reply:
@maddenf I didn't want to say all tests because many tests are intended to be run via ci tools against deployed artifacts. So maybe just: `Run the following scripts, ensuring no failures:`?

Reply:
Sounds good @jamesobrooks . Let's go with that.

Reply:
@mmarcotte, I updated it.